beacon-chain/core/peerdas/helpers.go

```diff
@@ -23,10 +23,13 @@ const (
 
 	// Number of cells in the extended matrix
 	extendedMatrixSize = fieldparams.MaxBlobsPerBlock * cKzg4844.CellsPerExtBlob
+
+	// Maxmimum value of an uint64
+	maxUint64 = ^uint64(0)
```

Comment:
we should use the value from the std lib (math.MaxUint64)

Reply:
Fixed in dc02b04dfe5524be4242ed7f47f3f653f06d8c40.